config/initializers/schedule_parameters.rb

```diff
@@ -0,0 +1,6 @@
+#sanitize the OSEM_SCHEUDLE_CELL_SIZE to be used for EventType::LENGTH_STEP
```

Comment:
I think we do, since it makes the system a bit more foolproof, we should not trust the user supplied data.